### ISSUE TITLE
 Add-TeamViewerCustomization , not working?

### ISSUE THREAD

**Issue by gonzague** [teamviewer/TeamViewerPS#62]
Hello,

In the docs, there is a mention of the command [Add-TeamViewerCustomization](https://github.com/teamviewer/TeamViewerPS/blob/main/Docs/Help/Add-TeamViewerCustomization.md) , however when trying to run the command on a machine it reports an error 👍 
```
Add-TeamViewerCustomization : Le terme «Add-TeamViewerCustomization» n'est pas reconnu comme nom d'applet de commande, fonction, 
fichier de script ou programme exécutable. Vérifiez l'orthographe du nom, ou si un chemin d'accès existe, vérifiez que le chemin 
d'accès est correct et réessayez.
Au caractère Ligne:1 : 1
+ Add-TeamViewerCustomization -Id 'xxxxx' -RestartGUI -RemoveExisting
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (Add-TeamViewerCustomization:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException
```

Is this expected ? 

thanks!

**Comment by ChristianJ-TV** [comment]
Hello @gonzague,

we a have special situation currently, which might be little confusing, as we are preparing the major version 2.

**Current Situation:**
On Github you are getting the version 1.5.2 as default package, but you could also get the new version 2 already, if you just download the source code.
On Powershell Gallery, you are also getting the version 1.5.2.

I assume, you installed TeamViewerPS from Gallery and got version 1.5.2.
If you now check the documentation on Github, you already see the upcoming cmdlets which are not included in version 1.5.2, like Add-TeamViewerCustomization.

**You options**
You can download and install the module from Github directly (not as package), or wait until we officially released it to Powershell Gallery.

**Comment by gonzague** [comment]
Ok thanks a lot @ChristianJ-TV  ! Maybe it would make sense in the docs to mention the minimum version necessary :-]

Thanks for the great work anyway it will be helpful
